.github/pr-title-checker-config.json

```
@@ -4,7 +4,7 @@
 		"color": "B60205"
 	},
 	"CHECKS": {
-		"regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Revert|i18n):)|(?:Bump) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)",
+		"regexp": "^(?:(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Revert|i18n):)|(?:Bump)) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)",
```

```suggestion
		"regexp": "^(?:(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Revert|i18n):)|(?:Bump)) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)$",
```

We should also make sure nothing sneaks in after Release x.y.z and Merge master into develop 😬

---

that's true for `Relase x.y.z` but not for `Merge master ..` as you can at https://github.com/RocketChat/Rocket.Chat/pull/24363

---

Ah .. well then we can use three separate groups, but tbh release ones will only be internal and that too not in hundreds, so doesn't matter ig